src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,34 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        final int date = 0;
+        final int hours = 2;
+        final int hourlyPay = 3;
+        final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
[Common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/exceptions/salary-info)
If you create a [formatter](https://docs.oracle.com/javase/tutorial/datetime/iso/format.html), make it a constant field.

Same for your int variables. And mame should be like DATE_INDEX - not simple DATE. Because it's not a date.